grep-0011-scheduler.md

```diff
@@ -0,0 +1,124 @@
+# GREP 0011 -- Create new scheduler to increase performance 
+
+- Original Author: David Hofstee <gnuradio@c001.nl>
+- Champion: David Hofstee <gnuradio@c001.nl>
+- Status: Draft 
+
+History:
+- 15-Feb-2018: Initial Draft
+
+## Abstract
+
+Recent presentations at GRCon showed that performance of Gnuradio can
```

Comment:
Since this is an important GREP, let's please stick to the official spelling: "GNU Radio"